projects/web-filter/src/pages/Popup.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+import "./Popup.css";
+
+export default function() {
+  useEffect(() => {
+    console.log("Hello from the popup!");
+  }, []);
+
+  return (
+    <div>
+      <img src="/coffee-filter.png" />
+      <h1>Web Filter</h1>
+      <p>by. Team Seoee - Interactive Study</p>
+    </div>
+  )
```

Comment:
```suggestion
import { useEffect } from 'react';

import './Popup.css';

export default function () {
  useEffect(() => {
    console.log('Hello from the popup!');
  }, []);

  return (
    <div>
      <img src="/coffee-filter.png" />
      <h1>Web Filter</h1>
      <p>by. Team Seoee - Interactive Study</p>
    </div>
  );
```

이것도 따옴표랑 import-sort 안 먹은 듯 합니다

Reply:
해주세요

Reply:
생푸쉬함요~